src/executables/unload_schema.c

```diff
@@ -3157,7 +3286,14 @@ emit_partition_info (print_output & output_ctx, MOP clsobj)
       return;
     }
 
-  output_ctx ("\nALTER CLASS %s%s%s ", PRINT_IDENTIFIER (name));
+  class_name_copy = strdup (name);
+  owner_name_p = strtok_r (class_name_copy, ".", &save_token);
+  class_name_p = strtok_r (NULL, ".", &save_token);
+
```

Comment:
```suggestion
  class_name_copy = strdup (name);
  owner_name_p = strtok_r (class_name_copy, ".", &save_token);
  class_name_p = strtok_r (NULL, ".", &save_token);
```
It looks better to replace a macro for repeated routine.

Reply:
I will change it to a macro.